liveview-android/src/main/java/org/phoenixframework/liveview/domain/base/ComposableView.kt

```diff
@@ -270,6 +272,17 @@ abstract class ComposableBuilder {
         }
     }
 
+    /**
+     * Tag used during the UI tests. It must be unique in the UI tree.
+     * ```
+     * <Composable test-tag="myTag" />
+     * ```
+     * @param testTag tag used during the UI tests.
+     */
+    private fun testTag(testTag: String) = apply {
+        modifier = modifier.then(Modifier.testTag(testTag))
+    }
+
```

Comment:
Is this restricted to just in the tests, or is this available for anyone to add?

Reply:
It's available to anyone. It can be useful to write tests for a client app.
https://developer.android.com/reference/kotlin/androidx/compose/ui/Modifier#(androidx.compose.ui.Modifier).testTag(kotlin.String)

Reply:
@bcardarella or @BrooklinJazz might be able to clarify, but I think Elixir developers would prefer to use the LiveView testing tools for their templates?

Also, if this is a modifier it should probably be done via a stylesheet. From what I understand, modifier order is important in Compose which can't be achieved via attributes.

Reply:
@carson-katri makes sense. But if we introduce a LiveView in an existing Android app, it would be important to write tests on the app side. Test tag will help to achieve this. 
You're right, the order of the modifiers is important. So my idea is that developers must declare the attributes in the right order (as they have to do in the native Android development).

Reply:
In this case, I think it's fine as an attribute, because it's more of a tag on a composable as oppossed to something more style-related (we allow `id="..."` and `tag="..."` in the SwiftUI client, even though they are modifiers).

However, stylesheets were added specifically for modifiers, and I don't think the Jetpack client should deviate here. The reasoning (at least regarding SwiftUI) was:

1. Modifiers can have a particular order, which is possible with attributes, but unclear
2. The same modifier can appear multiple times in a chain, whereas attributes can only be used once per-element:
```swift
Text("")
  .padding(5)
  .background(.red)
  .padding(10)
```
```html
<!-- only one of the padding values is returned -->
<Text padding="5" background="red" padding="10" />
```
3. Modifiers can take complex values, and some have multiple arguments. For example, this cannot be clearly represented in an attribute: `stroke(.red, style: StrokeStyle(lineWidth: 10, dash: [1, 3]))`
4. Modifiers can have more content inside of them, which is not possible for attributes. In this case, we use `template="..."` to find the content.

I believe most of these points will apply to Compose modifiers as well. One example I see in the Compose docs is `Modifier.draggable`. This seems nearly impossible to represent as an attribute.

Reply:
Agreed. Please, open an issue to handle the modifiers chain. I believe this must be a priority for me instead of working on new components. 
@bcardarella @carson-katri I will need help to understand how to do this for Jetpack Compose.